Revise the root of LittleSkin and add tips in the faq.md (#15)
LittleSkinCommspt/LittleSkinManualDev: en/faq.md

[[toc]]

## Site Using

### Which domain should I use?

> LittleSkin has three domains in service:  `mcskin.littleservice.cn`, `littleskin.cn`, and `littlesk.in`. They provide the same services.

Reviewer: 这里的标题是 `我应该使用哪个域名？`。作为回答，你只说了有三个域名，而且功能一样，而没有解答问题。

Author: We suggest using littleskin.cn

If it is still not working, please join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) and contact the administrator of LittleSkin.

### Invalid Cape file

LittleSkin has a strict requirement with the cape. Plases make the cape under the official standard of Minecraft.

LittleSkin doesn't support the cape with the format of 22\*17. Plaess change the format into 64\*32.

### Invaild Error

If you get a `Invaild Error` when you are using LittleSkin, please send the snapshot of the error and join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) and contact the administrator of LittleSkin.

### When will LittleSkin hold events? What's the activities in the events?

The events will be held from time to time. The details of the events will be announced by the administrator of LittleSkin after discussion.

Most of the time, LittleSkin will hold events on big festival(such as The Spring Festival), the birthday of the LittleSkin in March or LittleSkin reach a new level of the user count.

### Is that possible for me to remove a ban if I have been banned?

> It's impossible. The ban is permanent, unless you have been banned by mistake. You will not be able to remove a ban even if you donate a large amount and write a written guarantee, 

Reviewer: ```suggestion
It's impossible. The ban is permanent unless you have been banned by mistake. You will not be unbanned even if you donate a large amount and write a written guarantee, 
```

### I can sees my skin in single player game but not the muilt players game?

It often happens when you are playing on a server which is using Yggdrasil to load the authentication server.

Because of the skin loading system in Minecraft, in muilt player games, the server which opens the online mode can load the players' skins successfully. If the server you are playing load `authlib-injector` and set up LittleSkin's Yggdrasil API correctly, you can load your skin.

If you need to load the players' skin in server, and you are the owner of the server, please set up `authlib-injector`, or you have to load your skins by Skin Mod.

### Why I can't use Universal Skin Mod to load my skin?

For some technical reasons, LittleSkin stop the support of UniSkinAPI in January 2020. You can still use the legacy way to load your skin, but we suggest to use CustomSkinLoader instead of Universal Skin Mod.

### The Launcher can't download authlib-injector.

Most of the time it is caued by your network connection. Change the mirror of the download source in the launcher.

> If you are using HMCL 3 and it doesn't work after changing the mirror of the download source in the launcher. Please download   authlib-injector, put it into `%appdata%\.hmcl\` for Windows or `$HOME/.hmcl/` for Linux and macOS. Rename it into `authlib-injector.jar`. You need to contact the launcher makers if you are using other launchers.

Reviewer: ```suggestion
If you are using HMCL 3 and it doesn't work after changing the mirror of the download source in the launcher. Please download the authlib-injector, put it into `%appdata%\.hmcl\` for Windows or `$HOME/.hmcl/` for Linux and macOS. Rename it to `authlib-injector.jar`. You need to contact the launcher makers if you are using other launchers.
```

[[toc]]

## Site Using

### Which domain should I use.

LittleSkin has three service domains. `mcskin.littleservice.cn`, `littleskin.cn`, and `littlesk.in`. They provide the same services.

### Why I can't receive my verify tickets?

Please check the trash box in you Inbox. If you still can't find the tickets, please try to resend the verify tickets.

If it doesn't work, please check if you bind the correct email address with your account. You need to correct the email address if you made a mistake on it.

> If it still doesn't make sence, please join the [user group](/en/user-group.html) and contact the administrator of LittleSkin.

Reviewer: ```suggestion
If it is still not working, please join the [user group](/en/user-group.html) and contact the administrator of LittleSkin.
```

### Is that possible for me to get my player name back if it has been taken by others?

It's impossible. It's usual to have the same player name, it's unfair for you if we change the owner of the player name. So whether who you are, we won't provide the service of changing the owner of the player name.

If you have found that others have taken your player name, you can contact the owner of the player name. Maybe it's possible to get it back, and you can get a new friend as well. Most of the time they are easy to talk if you consult with them.

How to cantact? Do it yourself!

If you have the Minecraft Account form the Microsoft Corporation or Mojang Studio, and the player name is the one that has been taken by otheres, you can bind your [Minecraft Account with LittleSkin](/en/newbee/player.html#bind-a-mojang-account) and you can get the player name back.

### Can't bind your Minecraft Account?

For Mojang Account:
Please make sure that you are using the same email in both LittleSkin and Mojang Account. You can change your email address in the Profile Page.

For Microsoft Account:

> Up to now, we don't support the Microsoft Account. But we will support soon.  

Reviewer: ```suggestion
For now, we don't support binding a Microsoft Account. But we will add the support soon.  
```

### I can't load the web / pictures successfully.

> Most of the time, it is caused by your browser.

Reviewer: ```suggestion
It is usually because of your web browser.
```

Try `Ctrl + F5` to clean cache and refresh the page, if you still can't load the web successfully, maybe your browser is outdated.

Many outdated browser doen't support the new tech that LittleSkin is using in the web. You need to update the browser if it is outdated.

Some of the browsers may using the oudated layout engine. Maybe it still doesn't work after you update your browser. We suggest to use the latest [Google Chrome](https://www.google.com/chrome) or [Mozilla Firefox](https://www.mozilla.org/firefox/new) or Chromium based [Microsoft Edge (Chromium)](https://aka.ms/msedge)

> If it still doesn't make sence, please join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) and contact the administrator of LittleSkin.

Reviewer: ```suggestion
If it is still not working, please join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) and contact the administrator of LittleSkin.
```

为什么啊？？？？不正常工作不等于没有道理！！！

> ### Invalid Skin file

Reviewer: ```suggestion
### Invalid Cape file
```

This is a common question. Most of the time, it is caused by the reasons below:

1. You didn't set up and configure the skin mod correctly.
2. You have set up some mods(such as NonUpdate) that are interfering with the connection between the skin mod and LittleSkin.
3. Your skin is cached by your skin mod,
4. You are using the skin of Alex type, and you are playing Minecraft 1.8 or below.

The solution is:

1. Set up and configure the skin mod correctly. You can learn how to do it by reading [Getting Started - Mod Configurations](/en/newbee/mod.html),
2. Delete these mods, or let them bypass the connection between the skin mod and LittleSkin,
3. Try again later. If it still doesn't update, please try to clean the cache of the skin mod by deleting the cache folder:
    - The cache CustomSkinLoader is at `.minecraft/CustomSkinLoader/caches`；
    - The cache SkinPort is at `.minecraft/cachedImages`
4. Change the skin mod, or use the skin of Steve type:
    - For Minecraft 1.7.10，please use SkinPort to load your skin,
    - For the version below of Minecraft 1.7.10, please use the skin of Steve type.

> If you skin your problem is not caused by the reasons below, or it still doesn't work after tring the solutions, please read [How to report a problem](/en/report.html). Join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) with the logs of the skin mod, or buy the one-by-one support service.

Reviewer: ```suggestion
If you skin your problem is not caused by the reasons above, or it still doesn't work after tring the solutions, please read [How to report a problem](/en/report.html). Join the [user group](/en/user-group.html) or send [email tickets](/en/email.html) with the logs of the skin mod, or buy the one-by-one support service.
```

### I can sees my skin in single player game but not the muilt players game?

It often happens when you are playing on a server which is using Yggdrasil to load the authentication server.

Because of the skin loading system in Minecraft, in muilt player games, the server which opens the online mode can load the players' skins successfully. If the server you are playing load `authlib-injector` and set up LittleSkin's Yggdrasil API correctly, you can load your skin.

If you need to load the players' skin in server, and you are the owner of the server, please set up `authlib-injector`, or you have to load your skins by Skin Mod.

### Why I can't use Universal Skin Mod to load my skin?

> For some technical reasons, LittleSkin stop the service of UniSkinAPI from the January in 2020. You can still use the legacy way to load your skin, but we suggest to use CustomSkinLoader instead of Universal Skin Mod.

Reviewer: ```suggestion
For some technical reasons, LittleSkin stop the support of UniSkinAPI in January 2020. You can still use the legacy way to load your skin, but we suggest to use CustomSkinLoader instead of Universal Skin Mod.
```

Reviewer: Why this problem has NOT been addressed? /cc @Nofated095

### The Launcher can't download authlib-injector.

Most of the time it is caued by your network connection. Change the mirror of the download source in the launcher.

If you are using HMCL 3 and it doesn't work after changing the mirror of the download source in the launcher. Please download   authlib-injector, put it into `%appdata%\.hmcl\` for Windows or `$HOME/.hmcl/` for Linux and macOS. Rename it into `authlib-injector.jar`. You need to contact the launcher makers if you are using other launchers.

### AccessToken is invalid.

Please close the game and delete the account in the Launcher, and login again.

> 请先尝试退出游戏并在启动器中删除账号，然后重新登录。

Reviewer: ```suggestion

```
